### PR TITLE
Added a Number conversion(Decimal to Binary) code in java

### LINKED ISSUE #964: Number Conversion (Decimal To Binary)
can i start working on it ?

### PR DESCRIPTION
**Pull request by LovepreetSingh12**
Fixes #964

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- added Decimal_to_binary conversion
- 

#### Languages Used:
- java
- 

#### Files Added:
- Decimal_to_binary.java
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Number_Conversion/Java/Decimal_to_binary.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        n = sc.nextInt();   //Taking the decimal input
+
+        //Simultaneously storing the remainder when number divided by 2 in the string in reverse order
+        while(n > 0){
+            rem = n % 2;
+            Bin = rem + Bin;
+            n = n / 2;
+        }
+        System.out.println("Binary number:"+Bin);   //output
+    }
```
Comment (MastersAbh) [inline Number_Conversion/Java/Decimal_to_binary.java]: for n = 0, your code will give blank output, when it should give 0. Can you please update your code to handle the scenario ?

Comment (LovepreetSingh12) [reply Number_Conversion/Java/Decimal_to_binary.java]: Done sir.

File: Number_Conversion/Java/Decimal_to_binary.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    //Driver Code
+    public static void main(String[] args){
+        int n, rem;
+        String Bin = "";    //String to store the binary number
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter any decimal number:");
+        n = sc.nextInt();   //Taking the decimal input
+
+        if(n == 0){
```
Comment (MastersAbh) [inline Number_Conversion/Java/Decimal_to_binary.java]: space after if

File: Number_Conversion/Java/Decimal_to_binary.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            Bin = "0";
+        }
+
+        //Simultaneously storing the remainder when number divided by 2 in the string in reverse order
+        while(n > 0){
```
Comment (MastersAbh) [inline Number_Conversion/Java/Decimal_to_binary.java]: space after while

File: Number_Conversion/Java/Decimal_to_binary.java (hunk 2 above)
Comment (LovepreetSingh12) [reply Number_Conversion/Java/Decimal_to_binary.java]: Done sir. Sorry for the delay

**Pull request merged**